### ISSUE TITLE
Error with single time step

### ISSUE THREAD

**Issue by rlbyrne** [EoRImaging/eppsilon#153]
I am attempting to process a simulation that contains a single time step only, and the job is failing with this error message:
```
% Variable is undefined: DATA_DIFF_MEAN.
% Execution halted at: PS_KCUBE         1175
   /home/rbyrne/eppsilon/ps_core/ps_kcube.pro
%                      PS_POWER           20
   /home/rbyrne/eppsilon/ps_core/ps_power.pro
%                      PS_MAIN_PLOTS     272
   /home/rbyrne/eppsilon/ps_core/ps_main_plots.pro
%                      PS_WRAPPER        307
   /home/rbyrne/eppsilon/ps_wrappers/ps_wrapper.pro
```
My call is `ps_wrapper, '/safepool/rbyrne/fhd_outputs/fhd_rlb_process_uv_density_sims_Feb2023', 'sim_uv_spacing_10', /png, /plot_kpar_power, /refresh_ps, /uvf_input, pol_inc=["xx"], /no_evenodd`

**Comment by rlbyrne** [comment]
This was run on the `fix_edge_cases` branch.

**Comment by rlbyrne** [comment]
Closed by #154
